README.md

```diff
@@ -1,44 +1,53 @@
-# Implementing a full adder on a Quantum Computer (really meaningful title here)
-
-This serves as a homework template for CP1 2021 Homework3.
+# Implementing a full adder on a Quantum Computer and a phase shift measurement scheme
 
 ## Introduction
+This project includes the implementation of a Full Adder and Phase Shift Measurement Scheme on IBM's Quantum Computer using Qiskit.
 
-You should include a short introduction what the project is about. Don't forget to include references/attribution to anything you read (documentation) or use(code) from somewhere else. Make sure you have the rights to do so as well.
-
-## Setup
+The Full Adder uses the example of 1+1+1 = 3.
 
-A detailed outline how to get you project running.
+The Phase Shift Measurement Scheme is able to determine the phase that a qubit underwent after being split and recombined by a Hadamard gate. The example used is pi/3.
 
-e.g.
+## References 
```

Comment:
Important! Please add reference inline, so it is clear what statement / code is from which reference!